ansible/roles/master/tasks/localBuildInstall.yml

```diff
@@ -9,6 +9,7 @@
     - kube-scheduler
     - kube-controller-manager
     - kubectl
+    - kubelet
```

Comment:
(or I should read the summary instead of just commenting blindly like a fool, I'm going to delete all my previous comments)